calcom/cal.com
# fix: Adjust height of sidebar to fit on screen when top banner is visible

Adjusts the height of the sidebar to fit on screen and not be scrollable when top banner is visible. 

Fixes #8783 

https://www.loom.com/share/dcaee4eae95f42ae941e2ef2bfefd4e3

**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Go to the event-types page and observe the sidebar when a banner is visible on the top.
